cpp/benchmarks/io/orc/orc_reader_benchmark.cpp

```diff
@@ -124,7 +124,7 @@ void BM_orc_read_varying_options(benchmark::State& state)
             // Need to assume that an additional "overflow" stripe is present
             stripes_to_read.push_back(num_stripes);
           }
-          read_options.set_stripes(stripes_to_read);
+          read_options.set_stripes({stripes_to_read});
```

Comment:
`segments_in_chunk` returns a `std::vector<cudf::size_type>`, which is what `set_stripes` accepts. Wouldn't this be... ?
```suggestion
          read_options.set_stripes(std::move(stripes_to_read));
```